proto/src/rangeserver.proto

```diff
@@ -0,0 +1,27 @@
+syntax = "proto3";
+package rangeserver;
+
+message RangeId {
+    string keyspace_id = 1;
+    string range_id = 2;
+}
+
+message RangeKey {
+    RangeId range = 1;
+    bytes key = 2;
+}
+
+service RangeServer {
+    // Our Prefetch rpc accepts a PrefetchRequest and returns PrefetchResponse
```

Comment:
This comment is pretty redundant, doesn't tell you anything more than what the code itself does so I'd just remove it